src/objectTranslations/npe03__Recurring_Donations_Settings__c-de.objectTranslation

```diff
@@ -20,6 +20,11 @@
         <label>Autom. Erstellung Raten-Opportunity</label>
         <name>InstallmentOppAutoCreateOption__c</name>
     </fields>
+    <fields>
```

Comment:
@ptewson-sfdo I think you mentioned this in slack, but yeah - there should not be any RD translations in this PR, or really any translations beyond very specifically what was changed in this PR. The root issue might be what you mentioned with the branch naming - noting that this branch has some robot files that I also don't think are supposed to be here. You may have to fix the branch name first, ensure the branch is rebased off of the proper parent and then manually restore all of these incorrect files back to the state in master.